.github/workflows/test-platforms.yml

```diff
@@ -0,0 +1,342 @@
+          else
+            echo "Parsing of the request arguments failed"
+            exit 1
+          fi
+
+      # we post statuses manually as this does not run from a pull_request event
+      # https://developer.github.com/v3/repos/statuses/#create-a-status
+      - name: Create in-progress status
+        uses: octokit/request-action@v2.x
+        with:
+          route: 'POST /repos/${{ github.repository }}/statuses/${{ needs.pr-info.outputs.sha }}'
+          context: '${{ env.STATUS_NAME }} ${{ needs.pr-info.outputs.launch_args }}'
+          description: '${{ steps.generate_query.outputs.description }}'
+          state: pending
+          target_url: 'https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      # Fetch boot.iso and configiure its local location
+      - name: Fetch boot.iso if available for platform ${{ matrix.platform }}
+        id: boot_iso_for_platform
+        run: |
+          set -eux
+          BOOT_ISO_PATH="${{ github.workspace }}/${{ matrix.platform }}.boot.iso"
+          BOOT_ISO_URL="file://$BOOT_ISO_PATH"
+          if [ "${{ matrix.platform }}" == "daily-iso" ]; then
+            ${{ github.workspace }}/kickstart-tests/containers/runner/fetch_daily_iso.sh $GITHUB_TOKEN $BOOT_ISO_PATH
+            echo "boot_iso=\"bootIso\":{\"x86_64\":\"${BOOT_ISO_URL}\"}," >> $GITHUB_OUTPUT
+          elif [ "${{ matrix.platform }}" == "rawhide" ]; then
+            curl -L https://download.fedoraproject.org/pub/fedora/linux/development/rawhide/Server/x86_64/os/images/boot.iso --output $BOOT_ISO_PATH
+            echo "boot_iso=\"bootIso\":{\"x86_64\":\"${BOOT_ISO_URL}\"}," >> $GITHUB_OUTPUT
+          else
+            echo "Boot.iso for ${{ matrix.platform }} can't be fetched."
+            echo "boot_iso=" >> $GITHUB_OUTPUT
+          fi
+
+      # Configure location of installation repositories for the platform
+      # Also default boot.iso is defined by the value of urls.installation_tree
+      # of kstestParams event structure.
+      - name: Set installation tree for the platform
+        id: set_installation_urls
+        run: |
+          set -eux
+          if [ "${{ matrix.platform }}" == "rhel8" ]; then
+            echo "installation_tree=http://download.eng.bos.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.10.0/compose/BaseOS/x86_64/os" >> $GITHUB_OUTPUT
+            echo "modular_url=http://download.eng.bos.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.10.0/compose/AppStream/x86_64/os" >> $GITHUB_OUTPUT
+          elif [ "${{ matrix.platform }}" == "rhel9" ]; then
+            echo "installation_tree=http://download.eng.bos.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.4.0/compose/BaseOS/x86_64/os" >> $GITHUB_OUTPUT
+            echo "modular_url=http://download.eng.bos.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.4.0/compose/AppStream/x86_64/os" >> $GITHUB_OUTPUT
```

Comment:
It would be nice if the URLs are defined at one place and sourced to places like this one. Such URLs are already defined at multiple places in the repo, so when switching to a new minor version of RHEL, multiple places need to be modified. Just an idea for a future improvement, not blocking this PR.

Reply:
@jstodola @jkonecny12 https://github.com/rhinstaller/kickstart-tests/pull/1081